### PR TITLE
Added overheating check.

### PR DESCRIPTION
**Pull request by jmaha**
Separated name variable from serialNumberString for future editable-names.

### CODE REVIEW

File: Sources/CombustionBLE/Probe.swift [hunk 1]
```diff
@@ -85,7 +92,7 @@ public class Probe : Device {
     
     /// Pretty-formatted device name
```
Comment (jjohnstz) [inline Sources/CombustionBLE/Probe.swift:93]: Should we just remove this?  Since "name" logic will be inside the app rather than the framework?

**Review by jjohnstz — APPROVED** [review]
Looks good

File: Sources/CombustionBLE/Probe.swift (hunk 1 above)
Comment (jmaha) [reply Sources/CombustionBLE/Probe.swift:93]: Oh, good question. I think the `.name` call has to stay for API compatibility. I hadn't thought about the fact that the editable device name would be outside the framework.

Comment (jmaha) [reply Sources/CombustionBLE/Probe.swift:93]: Leaving it per discussion.

**Pull request merged**